backend/src/modules/users/controllers/users/users.controller.ts

```diff
@@ -46,4 +49,30 @@ export class UsersController {
     }
   }
 
+  @UseGuards(AuthGuard)
+  @Patch()
+  async editUser(
+    @Body() userDto: EditUserDto,
+    @Request() request: express.Request,
+    @Response() response: express.Response,
+  ) {
+    const userId: string = request.signedCookies[userIdCookieKey];
+    userDto.id = userId;
+    const validateResult = await this.usersService.validate(userDto);
```

Comment:
Validation of dtos is not business of services. If we will have enough time, we will try to insert json-schema wrapped in ts.
This is just note